src/main/java/com/trolley/trolley/InvoiceLine.java

```diff
@@ -0,0 +1,259 @@
+package com.trolley.trolley;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.trolley.trolley.types.Amount;
+
+public class InvoiceLine
+{
+    private String id;
+    private String invoiceLineId;
+    private String status;
+    private String description;
+    private Amount unitAmount;
+    private String quantity;
+    private Amount discountAmount;
+    private Amount taxAmount;
+    private Amount totalAmount;
+    private Amount dueAmount;
+    private Amount paidAmount;
+    private String externalId;
+    private boolean taxReportable;
+    private List<String> tags;
+    private InvoiceCategories category;
+    private boolean forceUsTaxActivity;
+
+    public static enum InvoiceCategories{
+        services,
```

Comment:
enum members should be uppercased, following Java naming conventions.

Reply:
Sure I could change that, wanted it to resemble our public API names to make it easy.